pandas_tutor/serialize.py

```diff
@@ -219,3 +226,57 @@ def serialize_scalar(val: Any) -> ScalarSpec:
 
 def pairs(seq: List[T]) -> List[Tuple[T, T]]:
     return [(seq[i], seq[i + 1]) for i in range(len(seq) - 1)]
+
+
+def truncate_val(
+    mark: List[Mark], before: EvalResult, after: EvalResult
+) -> Tuple[EvalResult, EvalResult]:
+    """
+    reduce the before and after EvalResults to simpler forms if possible
```

Comment:
```suggestion
    when we have really big dataframes / series, we'll trim out rows to avoid slowing down user's browsers.
    NOTE: this method mutates (!) the `before` and `after` objects.
```

(wrap the above line to 80 characters if necessary)